src/opm/parser/eclipse/share/keywords/000_Eclipse100/R/ROCKPAMA

```diff
@@ -3,6 +3,8 @@
   "sections": [
     "PROPS"
   ],
+  "size": "UNKNOWN",
+  "_comment_size": "It is tempting to use TABDIMS::NTROCC, but that has an untrivial default value",
```

Comment:
non

Reply:
ok. Is there a place where we could put this, or should we just hope noone else will insert NTROCC?

Reply:
i don't know but it's nontrivial, not untrivial.

Reply:
(yeah, a unit test would do..)